.github/workflows/import/poll-logs.js

```diff
@@ -207,26 +208,31 @@ const saveLivePaths = (livePaths) => {
 
 async function main() {
   await getImsToken();
-  const entries = await fetchLogsForSite(
+  const TO_PARAM = process.env.LAST_RUN_ISO_TO || new Date().toISOString();
+  if(localPathsToImport.length) console.log("Importing paths from local environment");
```

Comment:
OCD:
```suggestion
  if (localPathsToImport.length) console.log("Importing paths from local environment");
```

Reply:
Ughh. For some reason the linter doesn't scan the dotfiles such as `.github` ... I don't think we can overwrite that. Thanks for the manual linting hawk eyes though 😁